Highway.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
-
+#include "HighwayPatrol.h"
```

Comment:
- [ ] remove `#include`, forward declare instead.  
- [ ] `#include` in the cpp file.